### PR TITLE
Drop meas

### PR DESCRIPTION
**Pull request by dshen109**
* clean up style tests
* add a `drop_measurement` method

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ COPY test/ ./test/
 COPY fixtures/ ./fixtures/
 
 # Check code style and run static analysis along with tests
-RUN flake8 . && \
+# Ignore W605 errors (\* as invalid escape sequence) for sphinx doc builds.
```
Comment (shakefu) [inline Dockerfile:29]: I don’t see that in the docstring?

File: influx/__init__.py [hunk 2]
```diff
@@ -432,7 +454,7 @@ def _safe_request(self, *args, **kwargs):
         # Create the database
         try:
             db_resp = self.create_database(database)
-        except RequestException as err:
+        except RequestException:
```
Comment (shakefu) [inline influx/__init__.py:457]: Why this change here?

Comment (dshen109) [reply influx/__init__.py:457]: flake8 complained

File: Dockerfile (hunk 1 above)
Comment (dshen109) [reply Dockerfile:29]: https://github.com/AxiomExergy/influx-client/blob/master/influx/__init__.py#L672 this was the problem docstring
